cratedb_toolkit/cfr/jobstats.py

```
@@ -42,13 +43,28 @@
+deanonymize_sql: bool = False  # Added global flag for deanonymization
+decoder_dict_path: str
 
 
-def boot(address: DatabaseAddress, report_address: t.Optional[DatabaseAddress] = None):
+def boot(
+    address: DatabaseAddress, report_address: t.Optional[DatabaseAddress] = None,
+    anonymize_statements: bool = False, decoder_dict_file: t.Optional[str] = None,
+    deanonymize_statements: bool = False):
     # TODO: Refactor to non-global variables.
-    global stmt_log_table, last_exec_table, cursor, report_cursor, last_scrape, interval
+    global stmt_log_table, last_exec_table, cursor, report_cursor, last_scrape, interval, anonymize_sql, deanonymize_sql, decoder_dict_path
```

Ruff (0.8.2)

Line too long (139 > 120)

(E501)